ocs_ci/ocs/constants.py

```diff
@@ -114,14 +114,18 @@
 
 UPI_INSTALL_SCRIPT = "upi_on_aws-install.sh"
 
-DEFAULT_SECRET = 'rook-ceph-csi'
-DEFAULT_BLOCKPOOL = 'rbd'
-DEFAULT_SC_RBD = 'ceph-rbd'
+DEFAULT_CLUSTERNAME = 'ocs-storagecluster'
+DEFAULT_SECRET = 'rook-csi-rbd-node'
+DEFAULT_BLOCKPOOL = f'{DEFAULT_CLUSTERNAME}-cephblockpool'
 DEFAULT_SC_CEPHFS = "cephfs"
 DEFAULT_ROUTE_CRT = "router-certs-default"
 DEFAULT_NAMESPACE = "default"
 IMAGE_REGISTRY_RESOURCE_NAME = "cluster"
 
+# Default StorageClass
+DEFAULT_STORAGECLASS_CEPHFS = f'{DEFAULT_CLUSTERNAME}-ceph-rbd'
+DEFAULT_STORAGECLASS_RBD = f'{DEFAULT_CLUSTERNAME}-cephfs'
```

Comment:
@vasukulkarni declaration is wrong
we need this
DEFAULT_STORAGECLASS_CEPHFS = f'{DEFAULT_CLUSTERNAME}-cephfs'
DEFAULT_STORAGECLASS_RBD = f'{DEFAULT_CLUSTERNAME}-ceph-rbd'

Reply:
and thats why its failing :), thanks much!